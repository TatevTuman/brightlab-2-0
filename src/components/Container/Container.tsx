import React, { memo } from 'react'
import { Children } from '@types'
import styles from './Container.module.scss'

interface ContainerProps {
  children: Children
}

const Container: React.FC<ContainerProps> = props => {
  const { children } = props

  return <div className={styles.container}>{children}</div>
}

export default memo(Container)
