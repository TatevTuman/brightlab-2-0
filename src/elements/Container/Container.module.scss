.container {
  position: relative;
  width: 100%;
  max-width: 1440px;
  min-width: 320px;
  margin: 0 auto;
  padding: 2.25rem 2rem;
}
