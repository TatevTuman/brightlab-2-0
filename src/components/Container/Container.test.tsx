import React from 'react'
import renderer from 'react-test-renderer'
import Container from './Container'

describe('Container', () => {
  it('render()', () => {
    const tree = renderer
      .create(
        <Container>
          <div>child</div>
        </Container>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
