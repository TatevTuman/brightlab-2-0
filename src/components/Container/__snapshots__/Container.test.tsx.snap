// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container render() 1`] = `
<div
  className="container"
>
  <div>
    child
  </div>
</div>
`;
